### PR TITLE
Ensemble Module

### PR DESCRIPTION
**Pull request by QGN123**

### CODE REVIEW

File: paddlets/ensemble/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# !/usr/bin/env python3
+# -*- coding:utf-8 -*-
+
+"""
+ensemble
+"""
+from paddlets.ensemble.ensemble_forecaster_base import EnsembleForecasterBase
+from paddlets.ensemble.stacking_ensemble_forecaster import StackingEnsembleForecaster
+from paddlets.ensemble.weighting_ensemble_forecaster import WeightingEnsembleForecaster
```
Comment (kehuo) [inline paddlets/ensemble/__init__.py:9]: no new line at the end of file.

File: paddlets/pipeline/pipeline.py [hunk 2]
```diff
@@ -454,7 +454,7 @@ def load(cls, path: str, pipeline_file_name: str = "pipeline-partial.pkl", model
             Pipeline: The loaded pipeline.
         """
         if not os.path.exists(path):
-            raise_log(FileNotFoundError(f"file exist, path : {path}"))
+            raise_log(FileNotFoundError(f"path not exist, path : {path}"))
```
Comment (kehuo) [inline paddlets/pipeline/pipeline.py:457]: if this change is NOT related to ensemble, please submit it in a seprate PR.

**Review by yangs16 — APPROVED** [review]
